package.json

```diff
@@ -21,8 +21,10 @@
     "build": "zx ./scripts/build.cjs",
     "test": "jest",
     "test:coverage": "jest coverage",
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix .",
+    "lint:js": "eslint .",
+    "lint:js:fix": "eslint --fix .",
```

Comment:
please also add `lint` and `lint:fix` to run all in same time locally

Reply:
Added